tests/ui/previews.js

```diff
@@ -0,0 +1,74 @@
+var helpers = require('../lib/helpers');
+
+function testListPreviewsDontExist(test) {
+    test.assertExists('.app-list:not(.expanded)');
+    test.assertNotVisible('.app-list-app .preview');
+}
+
+casper.test.begin('Mobile previews on popular page', {
```

Comment:
these *could* be part of the app_list tests so that every app list page is tested. up to you